core/README.md

```diff
@@ -0,0 +1,31 @@
+# Chainlink Core
+
+This directory contains the majority of the code for the backend of Chainlink.
+
+Outside of the code contained in this repo, some static assets are pulled in using
+[packr](https://github.com/gobuffalo/packr), in the sibling directory `packr/`.
+For the backend, the solidity ABIs are used to generate types for interacting with
+Ethereum contracts. Additionally, the static assets generated by compiling the
+sibling directroy `operator-ui/` are built by packr and included in the final
+binary.
+
+See the [root README](https://github.com/smartcontractkit/chainlink/blob/master/README.md#install)
```

Comment:
I _think_ if you link to a relative path (`../README.md#install`), Github will fill in the rest for you with the current branch/commit selected.